content/about/resources.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Resources
+description: 'Resources | Node.js'
+authors: XhmikosR, mikeal, fhemberger, Fishrock123, yous, tomgco, tniessen, SMcCandlish, saadq, Trott, Gornstats, piperchester, naoufal, lpinca, j9t, bnoordhuis, harshadsabne, Chris911, MrJithil
+category: about
+---
+
+## Logo Downloads
+
+Please review the [trademark policy](/trademark) for information about permissible use of Node.js® logos and marks.
+
+Guidelines for the visual display of the Node.js mark are described in
+the [Visual Guidelines](/files/foundation-visual-guidelines.pdf).
```

Comment:
This link is broken.

Reply:
It's already amended. We may need to regenerate the preview.

Reply:
Will look into it after a new preview generation.